src/qt/rpcconsole.h

```diff
@@ -145,18 +145,18 @@ public Q_SLOTS:
     };
 
     interfaces::Node& m_node;
-    Ui::RPCConsole *ui;
-    ClientModel *clientModel;
+    Ui::RPCConsole *ui = nullptr;
```

Comment:
Same as above:
https://github.com/bitcoin/bitcoin/blob/727175a08dffcb5f18f8faac562ab54f89e5f36e/src/qt/rpcconsole.cpp#L454-L463

Reply:
@promag See the pull request description – it was only `editStatus` and `autoCompleter` that were previously neither initialized where defined nor in constructor. The other ones:
> Also; initialize non-static class members where they are defined in accordance with developer notes.

Reply:
Same as above
```cpp
Ui::RPCConsole* const ui;
```